src/context.ts

```diff
@@ -627,8 +630,9 @@ export class Context<
     headers?: HeaderRecord
   ): Response => {
     // Optimized
-    if (this.#isFresh && !headers && !arg && this.#status === 200) {
+    if (this.#isFresh && !headers && !arg) {
       return new Response(data, {
+        status: this.#status,
```

Comment:
I think `new Response()` without `status` is faster than `new Response()` with `status`. Try this script:

```ts
import { bench, group, run } from 'mitata'

group(() => {
  bench('without status code', () => {
    new Response('hi')
  })
  bench('with status code', () => {
    new Response('hi', {
      status: 200,
    })
  })
})

run()
```

On Node.js, it's almost the same, but on Bun and Deno, "without" is faster.

Reply:
Of course, there is overhead for the `if` statement to compare `this.#status` between `200`:

```ts
if (this.#isFresh && !headers && !arg && this.#status === 200) {}
```

But I think "Response without status" is more efficient.

Reply:
thanks!